### PR TITLE
Updated Readme and fixed minor bugs

### PR DESCRIPTION
**Pull request by kungfucat**
Fixed bug of drag view, changed the icon, updated Readme, and optimised reloading of graphs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,112 @@
...
-* Knowledge of JAVA
-* Some experience in developing Android apps (Hello-World App)
+> To get started...
+
+### Step 1
+- **Option 1**
+    - 🍴 Fork this repo!
+
+- **Option 2**
+    - 👯 Clone this repo to your local machine using `git clone https://github.com/utkarshmttl/eModules.git`
```
Comment (utkarshmttl) [inline README.md]: How does one contribute by cloning this repo directly?

Comment (kungfucat) [reply README.md]: Yeah right. Will remove it then.

Comment (utkarshmttl) [reply README.md]: If you want, can add an section like How to set up? and include this there? and also how to create a db and where to put it, but I think that can be done later when we actually remove the current db.

Comment (kungfucat) [reply README.md]: Will surely add this once the app is flexible enough to handle a different db.

Comment (utkarshmttl) [reply README.md]: Cool.

**Pull request merged**